test/references/show/ConnectivityTracer.txt

```diff
@@ -1 +1 @@
-ConnectivityTracer(1, 2, 3)
\ No newline at end of file
+ConnectivityTracer{BitSet}(2,)
```

Comment:
why has this changed?

Reply:
I added it for code coverage of `Base.show` 🤷‍♂️